### PR TITLE
Add end-user documentation

### PR DESCRIPTION
**Pull request by angcl**
Adds end-user documentation to the project. Please already have a look if my work goes into the right direction.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+![Wapuu Customizer](assets/images/customizer.png)
+
+Here in the customizer, you have different types of wearables, which you can use to dress your Wapuu. First of all you have some basic shapes, which differ in form and color. There is a section with things which the Wapuu can hold, like orbs or guitars. Furthermore you can give your little fellow a hat, a haircut or even a crown. Then you have a category with general accessiores like a superhero cape, a baguette or smart-looking glasses. Your Wapuu can also wear a sweater or a t-shirt. And finally you can also dress him with different shoes, which suit to the rest of their outfit.
+
+Some of the items are locked in the beginning and can be unlocked through pearls.
+
+![A locked item in the Customizer](assets/images/customizer_locked_item.png)
+
+When you click on a locked wearable, a modal will popup and ask you if you want to buy this item. On the top right you can find an information icon. When you hover over that item, you will see more information about the currently shown Wapuu. If you click on *OK", you confirm your purchase and therefore unlock the item.
```
Comment (Ancocodet) [inline docs/index.md:36]: The *OK" seems wrong, either *OK* or "OK" should be correct.

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+A green bar in the customizer shows you how much pearls currently are in your possession.
+
+![Pearls balance bar in Wapuu customizer](assets/images/pearls_balance.png)
+
+You can also name your little fellow. Above your Wapuu you can see a text field. Just change the standard Wapuu's name *Wapuu" into something you like more.
```
Comment (Ancocodet) [inline docs/index.md:42]: Same as *OK" (*Wapuu")

File: docs/index.md (hunk 1 above)
Comment (angcl) [reply docs/index.md]: Ty, fixed!

File: docs/index.md (hunk 2 above)
Comment (angcl) [reply docs/index.md]: Thanks, this is fixed as well.

**Review by DerHerrFeldmann — APPROVED** [review]

**Pull request merged**